rsp/00x_bz_rest_api.md

```diff
@@ -0,0 +1,159 @@
+# RSP - 1 — *bz* REST API
+
+## `POST /login`
+
+### Synopsis
+
+Use this method to authenticate.
+
+### Request body
+
+```json
+{
+    "email": "$(email: String)",
+    "passwd": "$(passwd: String)"
+}
+```
+
+### Response body
+
+```json
+{
+    "id": "$(id: Int)",
+    "infos": {
+        "email": "$(email: String)"
+    }
+}
+```
+
+### Response status
+
+Code | Description
+-----|-------------
+200  | Success
+400  | Error in request body
+403  | This account is already connected
+404  | No account for the given url
```

Comment:
Returning a different error code when an account does not exists and when an account exists but the password is incorrect leaks some information that an attacker could use to first find a correct account name, and then do password attacks on it. I would prefer if we just have an error code for "username or password incorrect". Github uses `401 Unauthorized` for this purpose.